README.md

```diff
@@ -147,6 +147,7 @@ Communications (Website, etc.)|Every other week on Monday at 10:00 PT|[Google Do
 End-User WG|Every other week on Thursday at 10:30 PT|[Google Doc](https://docs.google.com/document/d/1FqEo2PfHPP8iZX_W-C3YgXFUjhfwmAhoREUE8ZJPJfc/)||
 eBPF|Every week on Thursday at 10:30 PT|[Google Doc](https://docs.google.com/document/d/13GK915hdDQ9sUYzUIWi4pOfJK68EE935ugutUgL3yOw)|[Slack](https://cloud-native.slack.com/archives/C02AB15583A)|
 Agent Management WG|Every other week on Tuesday at 11:00 PT|[Google Doc](https://docs.google.com/document/d/19WA5-ex8rNFIBIyVb5VqMXfWNmUQwppGhN8zBeNG0f4)|[Slack](https://cloud-native.slack.com/archives/C02J58HR58R)|
+Prometheus Interoperability WG|Every other week on Wednesday at 08:00 PT|[Google Doc](https://docs.google.com/document/d/19bnXziPn2MZ9wO6684UoI4D-LCjGL5bTJkGhux29bx8/edit#)|[Slack](https://cloud-native.slack.com/archives/C01LSCJBXDZ)|
```

Comment:
@alolita can you please transfer ownership of the meeting notes doc to [cncf-opentelemetry-governance@lists.cncf.io](mailto:cncf-opentelemetry-governance@lists.cncf.io)?


see instructions in https://github.com/open-telemetry/community/issues/1147:

> ## Process to transfer ownership
>
> 1. Click the share button 
> 2. Invite cncf-opentelemetry-governance at lists.cncf.io to be an editor
> 3. Click the share button again
> 4. Click on the arrow next to the governance committee email and select "Transfer Ownership" 
> 5. Click "Send Invitation"

Reply:
@alolita please take a look 🙂

Reply:
going to merge this, will follow-up with @alolita